Content.Server/Resist/EscapeInventorySystem.cs

```diff
@@ -67,7 +69,8 @@ private void OnRelayMovement(EntityUid uid, CanEscapeInventoryComponent componen
         // Contested
         if (_handsSystem.IsHolding(container.Owner, uid, out _))
         {
-            AttemptEscape(uid, container.Owner, component);
+            var disadvantage = _contests.MassContest(container.Owner, uid, rangeFactor: 3f);
+            AttemptEscape(uid, container.Owner, component, disadvantage);
```

Comment:
```suggestion
            AttemptEscape(uid, container.Owner, component, _contests.MassContest(container.Owner, uid, rangeFactor: 3f));
```

Reply:
No, this makes it clear this is DISADVANTAGE and not the opposite. It's best for maintainability to keep it separate, because I had to fix this exact mistake four times across three forks: once on frontier, once on delta, before contests rework on EE, and now finally post-rework on EE.

Reply:
Fine, whatever.